src/components/Analysis/GameSummaryCard/GameSummaryCard.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { BigBoard } from '../../Board/BigBoard/BigBoard';
+import {
+  Card,
+  CardPrimaryAction,
+  CardActions,
+  CardActionButton,
+} from '@rmwc/card';
+import { formatDistanceStrict } from 'date-fns';
+import { FinishedGameState } from '../../../state/AppState';
+import { Typography } from '@rmwc/typography';
+import styles from '../../../views/Analysis/Overview/AnalysisOverview.module.css';
+import { ListDivider } from '@rmwc/list';
+import { Icon } from '@rmwc/icon';
+import { Link } from 'react-router-dom';
+
+interface GameSummaryCardProps {
+  gameNumber: number;
+  game: FinishedGameState;
+  link: { tag: typeof Link; to: string };
+}
+
+function getGameSummary(winner: 'X' | 'O' | null, moves: number) {
+  if (winner) {
+    return (
+      <>
+        <Icon
+          icon={{
+            icon: winner.toLowerCase(),
+            size: 'medium',
+          }}
+          className={styles.winnerIcon}
+        />
+        won after {moves} moves.
+      </>
+    );
+  }
+
+  return <>{moves} moves resulted in draw.</>;
+}
+
+export function GameSummaryCard(props: GameSummaryCardProps) {
```

Comment:
Could you make a first review of this component?
(Ignore the `style` tags for now, I'll replace them with proper css classes later.)